src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -179,7 +179,7 @@ public void initialize() {
   }
 
   public void onCloseButtonClicked() {
-    onCloseButtonClickedListener.run();
+    getRoot().setVisible(false);
```

Comment:
Don't think that is sufficent to effectively close it

Reply:
Previously this listener was created when the JFXDialog was created. I guess it was needed to make the dialog disappear properly / to trigger the close animation.
Now it just hides and reveals the create game window. It works in the same way in the mod and map details.
If you want to have it "more" closed I need you to go into more detail how you want to achieve that.
Don't know if it's worth the effort just to unload a few elements and one map preview picture from memory, though.